### ISSUE TITLE
pre-trained models

### ISSUE THREAD

**Issue by LCF2764** [SmoothKen/KKNet#1]
You've done a great job! Do you have any plans to release pre-trained models in the future?

**Comment by SmoothKen** [comment]
Yes, along with a more detailed README, once I survive through the current coding load.
